modules/ROOT/pages/cors-policy.adoc

```diff
@@ -4,24 +4,47 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: cors, api gateway, gateway, policy
 
-Cross-Origin Resource Sharing (CORS) is a mechanism by which a web application can access resources defined in another domain. Browsers for example, rely on this standard for allowing cross origin requests.
+[%autowidth.spread,cols="a,a"]
+|===
+>s| Policy name | cross-origin resource sharing (CORS)
+>s| Summary      | Enables access to resources residing in external domains.
+>s| Category | Compliance
+>s| Available from Mule version | v4.1.1
+.4+>.^s| Returned Status Codes |
```

Comment:
The status code that returns are missing right? 

@mtqp could you fill it in? This is one of the confusing aspects about CORS.

Reply:
Noted, will work with Mariano!